### DISCUSSION TITLE
add a note feature

### DISCUSSION THREAD

**Discussion by Harshjeet** [dailydotdev/daily#1569] [Feature Requests]
Hi,
Pls add a note feature so that it will boost our productivity with having markdown capabilities

**Comment by idoshamun** [comment]
Hi @Harshjeet,

What do you mean by "a note feature"? Please elaborate

**Comment by Harshjeet** [comment]
I mean if any article have some important key points ... then we can write
on it.

**Reply by idoshamun** [reply]
thanks for the clarification
